Makefile.PL

```diff
@@ -137,6 +137,19 @@ CONTRIBUTORS.txt:
 
 END_CONTRIBUTORS
 
+    my $docker_make = <<'END_DOCKER';
+
+docker-build: dist
+	sudo docker build --tag zonemaster/ldns:local --build-arg version=$(VERSION) .
+
+docker-tag-version: docker-build
+	sudo docker tag zonemaster/ldns:local zonemaster/ldns:$(VERSION)
```

Comment:
I agree that it doesn't look good when it's inconsistent. But this isn't the place to fix it. This is just how Zonemaster::LDNS version numbers are written everywhere. See https://metacpan.org/author/ZNMSTR for an example.

I guess maybe we should decide upon which version scheme to use (i.e. with or without the v). Personally I like it better without the v. The only practical difference I know of is that sometimes you need an extra step to strip the v because some tool doesn't allow it. I'm not sure if there are any tools that require the v, but if so it's easier to add characters than to remove them.